### PR TITLE
prepare chemcaption data

### PR DESCRIPTION
**Pull request by kjappelbaum**

### CODE REVIEW

File: data/tabular/chemcaption_rdkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,349 @@
+---
+name: chemcaption_rdkit
+description: |-
+    This dataset contains molecular descriptors, mostly derived using RDKit.
+targets:
+    - id: num_valence_electrons
+      type: categorical
```
Comment (MicPie) [inline data/tabular/chemcaption_rdkit/meta.yaml:7]: or `continuous`?

Comment (MicPie) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: with `significant_digits: 0`
not sure what is better?

File: data/tabular/chemcaption_rdkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+    - id: non_rotable_proportion
+      type: continuous
+      significant_digits: 3
+      description: proportion of non-rotatable bonds
+      names:
+          - noun: proportion of non-rotatable bonds
+          - noun: non-rotatable bond proportion
+    - id: num_single_bonds
+      type: continuous
+      significant_digits: 3
```
Comment (MicPie) [inline data/tabular/chemcaption_rdkit/meta.yaml:29]: can the `num_X_bonds` be even a float number?

File: data/tabular/chemcaption_rdkit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+      description: number of triple bonds
+      names:
+          - noun: fraction of triple bonds
+          - noun: triple bond fraction
+          - noun: triple bond proportion
+    - id: num_aromatic_bonds
+      type: continuous
+      significant_digits: 3
```
Comment (MicPie) [inline data/tabular/chemcaption_rdkit/meta.yaml:52]: same here, can this be a float number?

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 1 above)
Comment (MicPie) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: actually that makes a difference for the sampling logic path, but I would need to check, but should work with both setups

**Review by MicPie — APPROVED** [review]
lgtm

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 1 above)
Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: this was a naming issue 
it is actually a float, i.e. really the fraction

Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: same for the others

Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: but I'll double check this again, there was a bug in feature naming as they had non-unique names

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 3 above)
Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:52]: added now also a check that this is a float

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 2 above)
Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:29]: column name issue

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 1 above)
Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:7]: done

**Comment by kjappelbaum** [comment]
let's do not merge it. Upon checking, there are still more issues with the data.

File: data/tabular/chemcaption_rdkit/meta.yaml (hunk 2 above)
Comment (kjappelbaum) [reply data/tabular/chemcaption_rdkit/meta.yaml:29]: updated

**Pull request merged**